src/main/java/minijava/MJError.java

```diff
@@ -0,0 +1,13 @@
+package minijava;
+
+/** Basic error class in this project. */
+public class MJError extends Error {
```

Comment:
What's the rationale behind extending `Error` rather than `Exception` or `RuntimeException`?

> An Error is a subclass of Throwable that indicates serious problems that a reasonable application should not try to catch.